### ISSUE TITLE
Wildcard expansion for enrich fields in enrich policy

### ISSUE THREAD

**Issue by dnhatn** [elastic/elasticsearch#108476]
Currently, users must specify all enrich fields using their full names in the enrich policy. Introducing support for wildcard expansion in enrich fields would enhance the experience for users and customers, especially in scenarios involving many enrich fields.

No changes are required for Ingest and ESQL to adopt this change.

**Comment by nielsbauman** [comment]
Hey @dnhatn, thanks for reporting this. It looks like this might be a duplicate of https://github.com/elastic/elasticsearch/issues/99240. What do you think? If you agree, we could close this issue and maybe you could add a comment on that issue with the use case you're describing over here.

**Comment by dnhatn** [comment]
@nielsbauman Yes, both are similar. Hence, I am closing this issue in favor of https://github.com/elastic/elasticsearch/issues/99240. Thank you!
